recipes/gstreamer/build.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+# gstreamer expects libffi to be in lib64 for some reason.
+mkdir $PREFIX/lib64
+cp -r $PREFIX/lib/libffi* $PREFIX/lib64
```

Comment:
Yep, we are sure. We've been testing Qt5 internally for more than a month with this package.

Besides, there's no way to set the path to libffi in `configure`. I already checked :-)